src/components/ProjectPage/index.jsx

```diff
@@ -7,96 +7,86 @@ import Modal from "../shared/Modal";
 import Title from "../shared/Title";
 import Select from "../shared/Select";
 import BottomNavigator from "../shared/BottomNavigator";
-import { getDiffingResult } from "../../services/pages";
 
+import { getDiffingResult } from "../../../services/pages";
 import usePageListStore from "../../../store/projectPage";
 import usePageStatusStore from "../../../store/projectInit";
 
 function ProjectPage() {
-  const [isLoaded, setIsLoaded] = useState(true);
-  const [pageList, setPageList] = useState([]);
-  const [selectPage, setSelectPage] = useState("");
-  const navigate = useNavigate();
-  const { status, setStatus } = usePageStatusStore();
+  const [isLoaded, setIsLoaded] = useState(false);
+  const { status, setStatus } = usePageStatusStore(state => state);
   const { byPageIds } = usePageListStore();
+  const navigate = useNavigate();
+  const [targetPageId, setTargetPageId] = useState("");
```

Comment:
P1: 관련 함수끼리 묶어주시면 좋을 것 같습니다 :)

```suggestion
  const [isLoaded, setIsLoaded] = useState(false);
  const [targetPageId, setTargetPageId] = useState("");
  const { status, setStatus } = usePageStatusStore(state => state);
  const { byPageIds } = usePageListStore();
  const navigate = useNavigate();
```